main.go

```diff
@@ -159,25 +199,9 @@ func main() {
 	}
 	log.WithFields(log.Fields{"tags": defaultTags}).Infoln("Default Tags")
 
-	// Parse AWS_REGION environment variable.
-	if len(region) == 0 {
-		region, _ = getMetadataRegion()
-		log.WithFields(log.Fields{"region": region}).Debugln("ec2Metadata region")
-	}
-	ok, err := regexp.Match(regexpAWSRegion, []byte(region))
-	if err != nil {
-		log.Fatalln("Failed to parse AWS_REGION:", err.Error())
-	}
-	if !ok {
-		log.Fatalln("Given AWS_REGION does not match AWS Region format.")
-	}
-	awsSession = createAWSSession(region)
-	if awsSession == nil {
-		err = fmt.Errorf("nil AWS session: %v", awsSession)
-		if err != nil {
-			log.Println(err.Error())
-		}
-		os.Exit(1)
+	if copyLabelsString != "" {
+		copyLabels = strings.Split(copyLabelsString, ",")
```

Comment:
Did you mean to call `parseCopyLabels` here?

Reply:
yes 🤦